.github/workflows/backup-production.yml

```diff
@@ -0,0 +1,16 @@
+name: production
```

Comment:
Maybe change the name of the workflow? It's going to have the same name as the deploy production workflow in the GitHub Actions tab.

Reply:
I intended it to go there since it's affecting the production Firebase but you're right about it and that it'd clutter the prod deployment. I'll rename it to something unique, perhaps as `backup`!

Reply:
I don't mind keeping it in the "production" workflow then, since it is related to the production. Although `backup` is good too!